src/vendor/composer/autoload_real.php

```diff
@@ -2,54 +2,37 @@
 
 // autoload_real.php @generated by Composer
 
-class ComposerAutoloaderInit30c84c47a44576eaa4b38cc478d5e11b {
-
-	private static $loader;
-
-	public static function loadClassLoader( $class ) {
-		if ( 'Composer\Autoload\ClassLoader' === $class ) {
-			require __DIR__ . '/ClassLoader.php';
-		}
-	}
-
-	/**
-	 * @return \Composer\Autoload\ClassLoader
-	 */
-	public static function getLoader() {
-		if ( null !== self::$loader ) {
-			return self::$loader;
-		}
-
-		require __DIR__ . '/platform_check.php';
-
-		spl_autoload_register( array( 'ComposerAutoloaderInit30c84c47a44576eaa4b38cc478d5e11b', 'loadClassLoader' ), true, true );
-		self::$loader = $loader = new \Composer\Autoload\ClassLoader();
-		spl_autoload_unregister( array( 'ComposerAutoloaderInit30c84c47a44576eaa4b38cc478d5e11b', 'loadClassLoader' ) );
-
-		$useStaticLoader = PHP_VERSION_ID >= 50600 && ! defined( 'HHVM_VERSION' ) && ( ! function_exists( 'zend_loader_file_encoded' ) || ! zend_loader_file_encoded() );
-		if ( $useStaticLoader ) {
-			require __DIR__ . '/autoload_static.php';
-
-			call_user_func( \Composer\Autoload\ComposerStaticInit30c84c47a44576eaa4b38cc478d5e11b::getInitializer( $loader ) );
-		} else {
-			$map = require __DIR__ . '/autoload_namespaces.php';
-			foreach ( $map as $namespace => $path ) {
-				$loader->set( $namespace, $path );
-			}
-
-			$map = require __DIR__ . '/autoload_psr4.php';
-			foreach ( $map as $namespace => $path ) {
-				$loader->setPsr4( $namespace, $path );
-			}
-
-			$classMap = require __DIR__ . '/autoload_classmap.php';
-			if ( $classMap ) {
-				$loader->addClassMap( $classMap );
-			}
-		}
-
-		$loader->register( true );
-
-		return $loader;
-	}
+class ComposerAutoloaderInit30c84c47a44576eaa4b38cc478d5e11b
```

Comment:
We need to exclude this folder `src/vendor/composer` from `format` and `lint`, pls check the composer.json